_posts/2020-10-04-newpipe-0.20.0.md

```diff
@@ -1,154 +1,176 @@
 <video src="/img/landscape_swipe_down.webm" poster="/img/0.20.0_thumbnail_swipe_down.png" class="img-responsive" autoplay loop muted controls></video>
-Since the player can now be minimized instead of closed, this means you get to retain your opened video queue! Let us go through an example: You watch a video, minimize it, then browse the trending page (you insane person), watch a video from there, quickly minimize it, browse for some calming music, and finally watch a video about pandas. As long as the minimized player isn't closed (by using the cross button or swiping down on it), you can tap back to go through those videos in reverse order! This allows you to browse a whole bunch of videos and quickly build up a video queue before you start watching them. You were able do this earlier, but you had to use the background or popup player queues as a workaround, since the main player's queue isn't accessible outside of the player.
+Minimizing the player instead of closing it means retaining your place in the video bread queue!
+Example: Be supposed to be doing something else, watch a video instead, minimize it, then browse _the trending page_ (you insane person). Watch some filth from there, then quickly minimize it, and shudder. Now browse for some calming music, and finally watch a video about pandas.
+
+If the player isn't closed (by using the cross button or swiping down on it), and instead stays minimized, you can tap "Back" to go through those videos in reverse order! You are making history, as the lead role of your own exploration. Pick videos for a quickly built video queue before watching.
+Before this was only possible by using the background or pop-up player.
 
 ### Thumbnail progress bar
 
-You can only see this when you play the video in the background or pop-up, but there is now a shiny new progress bar underneath the thumbnail! It updates in real time, gets a little counter and a red bar - it's great, really.  
+Bar progress, nail a thumb! If playing video in the background or in a pop-up, take note of the shiny progress bar underneath the thumbnail.
+It updates in real time, with a little counter and a red bar. The audacity.
 <img class="no-flow vertical" src="/img/screenshots/progress_bar.png"/>
 
-All of what you've read above was made possible by the lean and mean coding machine, the one and only - (drumroll) - [@avently](https://github.com/avently)! Seriously, you should all sacrifice your first born child in his name. Or just say thank you, whatever works out for ya (that would be kind of boring, though).
```

Comment:
This is just a blatant disregard for the attention of the reader. It is a video, it doesn't work at all in writing. There is again no level of sarcasm or irony established to pull off "you should all sacrifice your first born child in his name". I redid the intention, and it works. Decide what you want to say, don't make a full secondary clause in a sentence second-guessing it.

Reply:
The entire point of the joke here is the contrast between the extreme action of 'sacrificing' versus the very tame action of 'thanking'. We don't need to establish a background for that, and there is no second-guessing.